thumbservice/thumbservice.py

```diff
@@ -164,16 +158,16 @@ def key_exists(key):
         return False
 
 
-def frames_for_requestnum(reqnum, request, rlevel):
+def frames_for_requestnum(request_id, request, reduction_level):
     headers = {
         'Authorization': request.headers.get('Authorization')
     }
-    params = {'REQNUM': reqnum, 'RLEVEL': rlevel}
+    params = {'request_id': request_id, 'reduction_level': reduction_level}
     return get_response(f'{settings.ARCHIVE_API}frames/', params=params, headers=headers).json()['results']
 
 
 def rvb_frames(frames):
-    FILTERS = {
+    PRIMARY_OPTICAL_ELEMENTS = {
```

Comment:
this is really nitpicky, but this could probably stay as FILTERS or FILTERS_BY_COLOR or something, since it is just a mapping between filter color and filter code. You would still compare it to the primary optical element in the frame